tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/CreatedumpTests.cs

```diff
@@ -0,0 +1,343 @@
+// <copyright file="CreatedumpTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Tools.dd_dotnet.ArtifactTests;
+
+public class CreatedumpTests : ConsoleTestHelper
+{
+    private const string CreatedumpExpectedOutput = "Writing minidump";
+    private const string CrashReportExpectedOutput = "The crash might have been caused by automatic instrumentation";
+
+    public CreatedumpTests(ITestOutputHelper output)
+        : base(output)
+    {
+        // Those environment variables can be set by Nuke, and will impact the outcome of the tests
+        EnvironmentHelper.CustomEnvironmentVariables["COMPlus_DbgMiniDumpType"] = string.Empty;
+        EnvironmentHelper.CustomEnvironmentVariables["COMPlus_DbgEnableMiniDump"] = string.Empty;
+        EnvironmentHelper.CustomEnvironmentVariables["DD_INSTRUMENTATION_TELEMETRY_ENABLED"] = string.Empty;
```

Comment:
nit (it does the same thing, just closer to what we normally do

```suggestion
        SetEnvironmentVariable("COMPlus_DbgMiniDumpType", string.Empty);
        SetEnvironmentVariable("COMPlus_DbgEnableMiniDump", string.Empty);
        SetEnvironmentVariable("DD_INSTRUMENTATION_TELEMETRY_ENABLED", string.Empty);
```